libafl_bolts/src/intel_pt.rs

```diff
@@ -0,0 +1,1022 @@
+//! Intel Processor Trace (PT)  low level code
```

Comment:
Wouldn't that fit better into `libafl_targets`?

Reply:
I'm completely open to move stuff around if there is a place that fits better, in general what should go into `bolts` and what in `targets`? I don't know much the latter crate

Reply:
targets is target-specificy stuff, i.e., instrumentation and bolts is just the very core foundational things.
In this case, interfacing with IntelPT might be bolts, but actually using it for coverage should go into targets. Maybe even its own crate?

Reply:
Got it thanks, as you prefer, even a new `libafl_intelpt` (?) crate is fine for me

Reply:
no strong opinion there, is there a usage for intel pt stuff out of target-specific things?

Reply:
I had a look at `libafl_targets` and if you don't mind I'd rather go with `libafl_intelpt`, this is not really instrumentation and I don't see any advantage in putting it there, there is no common code between the two

Reply:
`libafl_targets` could fit i think.
but since it seems to use specific dependencies, maybe it's better to move it separately.
otherwise we'll have to manage a bunch of dependencies behind a feature flag, it's a bit annoying.

Reply:
yeah, ok moving to `libafl_intelpt` then